proto/vendored/cosmos_sdk/tendermint/p2p/types.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+package tendermint.p2p;
+
+// This file contains types which are copied from
+// https://github.com/celestiaorg/cosmos-sdk/blob/v1.18.3-sdk-v0.46.14/proto/tendermint/p2p/types.proto
+
+message NetAddress {
```

Comment:
unused type?

Reply:
Removed in [83ac582](https://github.com/astriaorg/astria/pull/963/commits/83ac582dcfd052211270d5feaa828e288c5fad86).